docker-compose-tests.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3.6'
+services:
+  postgres:
+    image: postgres:11.2-alpine
+    restart: always
+    volumes:
+    - db_data:/var/lib/postgresql-test/data
+  graphql-engine:
+    image: hasura/graphql-engine:v1.0.0-beta.8
+    ports:
+    - 9000:8080
+    depends_on:
+    - postgres
+    restart: always
+    environment:
+      HASURA_GRAPHQL_DATABASE_URL: postgres://postgres:@postgres:5432/postgres
+      HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
+      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
+  hasura-service-tests:
+    build:
+      context: ./database-service
+      dockerfile: ./Dockerfile
+      network: host
```

Comment:
I am using the `host` network because building the Docker image needs internet connection, which is not enabled by default on the build server. In the past, in our saleor-based backend, I had to build the docker image with option `--network host`, which is exactly what that line is replicating.

Reply:
So here, as I said in a previous comment, the host's network is only used to **build** the image, not for any other purpose.